src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -370,6 +433,7 @@ public function addPaymentAction(int $orderId, Request $request): RedirectRespon
     private function getErrorMessages(OrderException $e)
     {
         return [
+            CannotEditDeliveredOrderProductException::class => $this->trans('Cannot edit product for delivered order', 'Admin.Orderscustomers.Notification'),
```

Comment:
I thought exceptions don't need translation domains, do they?

Reply:
It maps exception (logic error) to user understandable error which has to be translatable.

Reply:
@MatShir

Reply:
@sarjon, I haven't specified on the specs but the old behaviours when the order was considered as delivered. The edit and delete button were hidden. Would be possible to keep the same behaviours?

Reply:
I suggest _You cannot edit the cart once the order delivered_ in _Admin.Orderscustomers.Notification_.

Reply:
Done.